CMakeLists.txt

```diff
@@ -1546,18 +1546,9 @@ set(EVENT_INSTALL_CMAKE_DIR
 
 export(PACKAGE libevent)
 
-function(gen_package_config forinstall)
-    if(${forinstall})
-        set(CONFIG_FOR_INSTALL_TREE 1)
```

Comment:
So, `CONFIG_FOR_INSTALL_TREE` can be removed from `cmake/LibeventConfig.cmake.in`?

Reply:
I don't see why not. It's being installed to `${CMAKE_BINARY_DIR}`, similar to the other `.cmake` files

Reply:
But if we will remove `CONFIG_FOR_INSTALL_TREE` code, then `LIBEVENT_INCLUDE_DIRS` will be incorrect.

https://github.com/libevent/libevent/blob/89505f8513b8590e7a7700d481c20562d0726d54/cmake/LibeventConfig.cmake.in#L113-L147

P.S. Sorry for the delay.